Sources/Other/CollectionReuseViewManager.swift

```diff
@@ -40,6 +40,16 @@ public class CollectionReuseViewManager: NSObject {
     return view
   }
 
+  public func dequeue<T: UIView> (type: T.Type) -> T {
+    let queuedView = reusableViews[String(describing: type)]?.popLast() as? T
```

Comment:
Looks great. Just one concern that I have. Can you verify that `String(describing: type)` matches the identifier of `String(describing: type(of: view))` in `queue(view:)`. I am a little worried that it might not match. Would be best if you can include a test for this method as well.

Reply:
Yes, that was my concern too. It does match.
But personally, for identifier usage, I would prefer NSStringFromClass instead of String(describing:) since in rare cases such as nested classes, NSStringFromClass will return a unique string to identify the class and I think it's more reliable.
For example..

```Swift
class SomeClass {
    class SomeClass {
        
    }
}

print(NSStringFromClass(SomeClass.self))
print(NSStringFromClass(SomeClass.SomeClass.self))
// will print...
// __lldb_expr_67.SomeClass
// _TtCC14__lldb_expr_679SomeClass9SomeClass

print(String(describing: SomeClass.self))
print(String(describing: SomeClass.SomeClass.self))
// will print...
// SomeClass
// SomeClass
```